Packet: Add clc snippets,tags to worker_pool and clc snippets to controller (#355)
kinvolk/lokomotive: docs/configuration-reference/platforms/packet.md

| `certs_validity_period_hours`         | Validity of all the certificates in hours.                                                                                                                                    | 8760            | false    |
| `worker_pool`                         | Configuration block for worker pools. There can be more than one.                                                                                                             | -               | true     |
| `worker_pool.count`                   | Number of workers in the worker pool. Can be changed afterwards to add or delete workers.                                                                                     | 1               | true     |

> | `worker_pool.ssh_pubkeys`             | Optional list of SSH public keys for user `core`. If not provided then SSH public keys provided for controllers will be used for the worker pool.                             | []              | false    |

Reviewer: ```suggestion
| `worker_pool.ssh_pubkeys`             | Optional list of SSH public keys for user `core`. If not provided, the value of `ssh_pubkeys` is used.                             | []              | false    |
```

- Omit "then" after if.
- Reference the concrete knob whose value is used.
- Avoid future tense.

If you change this, please fix all similar occurrences.

| `worker_pool.clc_snippets`            | Worker Flatcar Container Linux Config snippets. If not provided then CLC Snippets provided for controllers will be used for the worker pool.                                  | []              | false    |

> | `worker_pool.tags`                    | Optional list of tags that will be propagated to nodes in the worker pool. If not provided then Tags for master nodes will be used for the worker pool.                       | -               | false    |

Reviewer: - Could you please fix the capitalization?
- We call controller nodes "controller nodes".
